main.py

```diff
@@ -0,0 +1,168 @@
+import sys
+class CategoryManager:
+
+    def __init__(self):
+        self.file_name = "CategoriesList.txt"
+
+    def check_file(self):
+         #check if file exists
+        try:
+            text = open(self.file_name, "r+")
+            print("Opened correctly.")
+            r_categories = text.read()
+            categories = r_categories.split( "\n" )
+            text.close()
+            return categories
+        except (IOError):
+            print("File not found.")
+
+    def categories(self):
+        pass
+
+    def add_category(self):
+        while True:
+            category_name = raw_input( "Enter a new category" )
+            if category_name in self.check_file():
+                print("Category already exist")
+            else:
+                #append file if category is added
+                cat = open(self.file_name,'a')
+                cat.write("\n")
+                cat.write(category_name)
+                cat.close()
+                print("Category is added")
```

Comment:
If category is added do we want to continue adding?

Reply:
nope